src/cmd/devdraw/cocoa-screen-metal.m

```diff
@@ -197,6 +197,7 @@ + (void)makewin:(NSValue *)v
 	[win setContentView:myContent];
 	[myContent setWantsLayer:YES];
 	[myContent setLayerContentsRedrawPolicy:NSViewLayerContentsRedrawOnSetNeedsDisplay];
+	[[NSNotificationCenter defaultCenter] addObserver:myContent selector:@selector(windowDidResize:) name:NSWindowDidResizeNotification object:nil];
```

Comment:
Should this pass `win` as the object? I wonder why `myContent` would work.

Reply:
I don't think we need to register this event for the view.  Just using the NSApplicationDelegate method should be enough.

Reply:
Thank you for your review!
@jxy If I use NSWindowDelegate method on AppDelgate,
How can I check inLiveResize flag of DevDrawView?